sklearn/impute.py

```diff
@@ -338,7 +338,7 @@ def _dense_fit(self, X, strategy, missing_values, fill_value):
                 most_frequent = np.empty(X.shape[0])
 
             for i, (row, row_mask) in enumerate(zip(X[:], mask[:])):
-                row_mask = np.logical_not(row_mask).astype(np.bool)
+                row_mask = np.logical_not(row_mask).astype(np.bool, copy=False)
```

Comment:
PEP8

Reply:
The code review capability of your editor must be adding extra spaces :)

This is 79 character long and valid according to PEP8.

Reply:
Ups got trapped by my own editor :)

Reply:
Actually we don't need `astype` here since `row_mask` will always be bool since `mask` is bool. However, there is a `astype` in fixes which might changed:
https://github.com/scikit-learn/scikit-learn/blob/73a55927cde7415e0db1b6b77cf9214df624a9cb/sklearn/utils/fixes.py#L321